### ISSUE TITLE
[Questions] any repo for uYouEnhanced for jailbreak

### ISSUE THREAD

**Issue by Mehranullah2017** [arichornlover/uYouEnhanced#560]
### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Is there an existing issue/question for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

✅ Yes, I believe this is a bug. I will open a new issue with the bug template

### My question

Please make a repo for your work because a lot of people have jailbreak nowadys
Thanks

**Comment by Mukul1127** [comment]
There is none

**Comment by bhackel** [comment]
Download and sideload with trollstore - most users are not jailbroken I’m pretty sure

**Comment by Mukul1127** [comment]
> Download and sideload with trollstore - most users are not jailbroken I’m pretty sure

That is true but the appeal of jailbreaking is that you don't have to patch the youtube app, just inject.

**Comment by bhackel** [comment]
Ah I forgot that we do not upload the IPA anymore, my apologies

**Comment by Mukul1127** [comment]
Its good

**Comment by PoloNX** [comment]
I hope someone will make a repo

**Comment by Mukul1127** [comment]
> I hope someone will make a repo

no chance until arichorn gets debs working, currently you can only build ipas
